src/sentry/utils/committers.py

```diff
@@ -229,6 +231,16 @@ def get_event_file_committers(project, event, frame_limit=25):
         {match for match in commit_path_matches for match in commit_path_matches[match]}
     )
 
+    logger.info(
+        "relevant_commits",
+        extra={
+            "relevant_commits": relevant_commits,
+            "project": project,
```

Comment:
@NisanthanNanthakumar What happens when you put in `project` and `group` as extra since these are instances of classes? Generally, we just log the IDs of the objects.

Reply:
@scefali probably don't need those fields

Reply:
@NisanthanNanthakumar I agree about the group but I definitely think the project is important if you are going to do debugging, though maybe not with these particular issue.

Reply:
@scefali true, I'll add it back there

Reply:
It probably get's `repr`d

Reply:
For now, I'm okay with the current fields in the logger. We can always add more later on.